### PR TITLE
Update uiuisms.rs

### PR DESCRIPTION
**Pull request by 44mira**
Added Average, Index Mask ('Where' in APL), GCD

### CODE REVIEW

File: site/src/uiuisms.rs [hunk 1]
```diff
@@ -117,6 +117,10 @@ uiuisms!(
     "/↧ [1 4 5 3 2]",
     /// Sort an array
     "⊏⌂. [1 4 5 3 2]",
+    /// Get the average of an array
+    "÷⇉⧻/+ [1 2 3]",
+    // Get the index mask of a bit array
+    "▽∶⇡⧻. [1 0 0 1]",
```
Comment (kaikalii) [inline site/src/uiuisms.rs:123]: This is actually present further down as "Find the indices of all 1s".
Maybe the description needs to be reworded to make it easier to search for?

File: site/src/uiuisms.rs [hunk 2]
```diff
@@ -173,6 +177,8 @@ uiuisms!(
     r#"▽×⍜'⊟⇌≡\↥.≠@ . "  abc xyz   ""#,
     /// Upscale a matrix
     "▽↯⧻,/÷△.⍉▽↯⧻,30 [0_1 1_0]",
+    /// Get the gcd of the top 2 values on the stack.
+    ";!(|2 ↬≠0.◿,⇉↥↧) 100 10",
```
Comment (kaikalii) [inline site/src/uiuisms.rs:181]: I'm hesitant to put a recursive function in here, as I don't think recursion is very "uiuic". The `recur` function is mostly there for completeness.
Is there a more array-y solution?

File: site/src/uiuisms.rs (hunk 1 above)
Comment (44mira) [reply site/src/uiuisms.rs:123]: Ah you're correct! My bad.

File: site/src/uiuisms.rs (hunk 2 above)
Comment (44mira) [reply site/src/uiuisms.rs:181]: I will look into it but as of the moment, Euclid's method is the only way I know how to get a GCD algorithmically.

**Pull request closed without merging**